### PR TITLE
Add npm audit as first step of build process

### PR DESCRIPTION
**Pull request by mattystone**
# Description

Please explain briefly the changes you made here.

## Checklist

- [ ] 3 Amigos
- [x] Acceptance Criteria met
- [x] Branch named {feature|hotfix|task}/{S4-*}
- [x] Commit messages are meaningful
- [x] Manually tested
- [ ] All unit tests passing
- [ ] WAVE accessibility testing tool ran on new or updated pages
- [x] Pulled latest master into feature branch
- [ ] Code reviewed
- [ ] New Docker environment variables are consistent with those on the Rebel1 environment
- [ ] If your pull request depends on any other, please link them in the description

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,6 +12,7 @@ clean:
 	rm -f ./build.log
 
 package-install:
+	npm audit --audit-level=moderate
```
Comment (mattystone) [inline Makefile:15]: Adding this at the initial stage of the build, ensures fast failure on issues and smallest threat window with vulnerable or compromised packages.

**Review by mateuszc-kainos — APPROVED** [review]

**Review by peadarkelly — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (peadarkelly) [reply Makefile:15]: Wait, how is it able to scan dependencies and sub-dependencies before they are installed?

Comment (matthewdoreilly) [inline Makefile:15]: We have an npm config file (`.npmrc`) which has the audit-level set to moderate.
Will the CI not pick that up, so we can just do `npm audit` in the Makefile instead?

Comment (matthewdoreilly) [reply Makefile:15]: Yeah I don't think npm audit will work if there's no package-lock file / node_modules

Comment (mattystone) [reply Makefile:15]: See: https://ci.platform.aws.chdev.org/teams/team-tiny-rebels/pipelines/dissolution-web/jobs/pull-request-analysis/builds/274

Passed CI build with functional report showing found 70 low severity vulnerabilities in 1571 scanned packages and no >=moderate vulnerabilities.

Local testing shows anything >= to audit level fails build.

Comment (mattystone) [reply Makefile:15]: You are right, i'll remove from here to avoid duplication.

**Review by matthewdoreilly — APPROVED** [review]

**Pull request merged**